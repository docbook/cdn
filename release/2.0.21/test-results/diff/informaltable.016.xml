<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: informaltable.016</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Unit Test: informaltable.016</h2>
            <div class="author">
               <h3>
                  <span class="personname">
                     <span class="firstname">Adam</span>
                     <span class="surname">Di Carlo</span>
                  </span>
               </h3>
            </div>
         </header>
         <pre deltaxml:deltaV2="A" class="screen language-none line-numbers">
            <code>+........+........+........+........+
|   A    |   B    |   C    |   D    |
+........+........+........+........+
|   E    |                         F|
+........+........+........+........+</code>
         </pre><div deltaxml:deltaV2="B" class="content">
            <pre class="screen language-none line-numbers">
               <code>+........+........+........+........+
|   A    |   B    |   C    |   D    |
+........+........+........+........+
|   E    |                         F|
+........+........+........+........+</code>
            </pre>
            <p>Spanspec test.  Cell "F" should be right aligned and spanning
the last 3 columns.</p>
            <figure id="informaltable" class="informaltable-wrapper">
               <div class="informaltable">
                  <table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                     <colgroup>
                        <col class="tcol1" align="center" width="25%"/>
                        <col class="tcol2" align="center" width="25%"/>
                        <col class="tcol3" align="center" width="25%"/>
                        <col class="tcol4" align="center" width="25%"/>
                     </colgroup>
                     <tbody>
                        <tr>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">A</td>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">B</td>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">C</td>
                           <td style="border-bottom: 1px solid ; " align="center">D</td>
                        </tr>
                        <tr>
                           <td style="border-right: 1px solid ; " align="center">E</td>
                           <td style="border-right: 1px solid ; " colspan="3" align="right">F</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </figure>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">Spanspec test.  Cell "F" should be right aligned and spanning
the last 3 columns.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="informaltable" class="informaltable-wrapper">
            <div class="informaltable">
               <table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                  <colgroup>
                     <col class="tcol1" align="center" width="25%"/>
                     <col class="tcol2" align="center" width="25%"/>
                     <col class="tcol3" align="center" width="25%"/>
                     <col class="tcol4" align="center" width="25%"/>
                  </colgroup>
                  <tbody>
                     <tr>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">A</td>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">B</td>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">C</td>
                        <td style="border-bottom: 1px solid ; " align="center">D</td>
                     </tr>
                     <tr>
                        <td style="border-right: 1px solid ; " align="center">E</td>
                        <td style="border-right: 1px solid ; " colspan="3" align="right">F</td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </figure>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>