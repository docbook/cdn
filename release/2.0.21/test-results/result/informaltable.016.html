<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for informaltable.016.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: informaltable.016.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">5</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: informaltable.016&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: informaltable.016&lt;/h2&gt;</span>
            <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Adam&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Di Carlo&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;pre <span class="att">class="screen language-none line-numbers"</span>&gt;
            <span class="tag">&lt;code&gt;+........+........+........+........+
|   A    |   B    |   C    |   D    |
+........+........+........+........+
|   E    |                         F|
+........+........+........+........+&lt;/code&gt;</span>
         &lt;/pre&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;Spanspec test.  Cell "F" should be right aligned and spanning
the last 3 columns.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;figure <span class="att">class="informaltable-wrapper"</span> <span class="att">id="informaltable"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="informaltable"</span>&gt;
               <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "</span>&gt;
                  <span class="tag">&lt;colgroup&gt;
                     <span class="tag">&lt;col <span class="att">align="center"</span> <span class="att">class="tcol1"</span> <span class="att">width="25%"</span>/&gt;</span>
                     <span class="tag">&lt;col <span class="att">align="center"</span> <span class="att">class="tcol2"</span> <span class="att">width="25%"</span>/&gt;</span>
                     <span class="tag">&lt;col <span class="att">align="center"</span> <span class="att">class="tcol3"</span> <span class="att">width="25%"</span>/&gt;</span>
                     <span class="tag">&lt;col <span class="att">align="center"</span> <span class="att">class="tcol4"</span> <span class="att">width="25%"</span>/&gt;</span>
                  &lt;/colgroup&gt;</span>
                  <span class="tag">&lt;tbody&gt;
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td <span class="att">align="center"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;A&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">align="center"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;B&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">align="center"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;C&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">align="center"</span> <span class="att">style="border-bottom: 1px solid ; "</span>&gt;D&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td <span class="att">align="center"</span> <span class="att">style="border-right: 1px solid ; "</span>&gt;E&lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">align="right"</span> <span class="att">colspan="3"</span> <span class="att">style="border-right: 1px solid ; "</span>&gt;F&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                  &lt;/tbody&gt;</span>
               &lt;/table&gt;</span>
            &lt;/div&gt;</span>
         &lt;/figure&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: informaltable.016&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: informaltable.016&lt;/h2&gt;</span>
            <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Adam&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Di Carlo&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;pre <span class="att">class="screen language-none line-numbers"</span>&gt;
               <span class="tag">&lt;code&gt;+........+........+........+........+
|   A    |   B    |   C    |   D    |
+........+........+........+........+
|   E    |                         F|
+........+........+........+........+&lt;/code&gt;</span>
            &lt;/pre&gt;</span>
            <span class="tag">&lt;p&gt;Spanspec test.  Cell "F" should be right aligned and spanning
the last 3 columns.&lt;/p&gt;</span>
            <span class="tag">&lt;figure <span class="att">class="informaltable-wrapper"</span> <span class="att">id="informaltable"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="informaltable"</span>&gt;
                  <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "</span>&gt;
                     <span class="tag">&lt;colgroup&gt;
                        <span class="tag">&lt;col <span class="att">align="center"</span> <span class="att">class="tcol1"</span> <span class="att">width="25%"</span>/&gt;</span>
                        <span class="tag">&lt;col <span class="att">align="center"</span> <span class="att">class="tcol2"</span> <span class="att">width="25%"</span>/&gt;</span>
                        <span class="tag">&lt;col <span class="att">align="center"</span> <span class="att">class="tcol3"</span> <span class="att">width="25%"</span>/&gt;</span>
                        <span class="tag">&lt;col <span class="att">align="center"</span> <span class="att">class="tcol4"</span> <span class="att">width="25%"</span>/&gt;</span>
                     &lt;/colgroup&gt;</span>
                     <span class="tag">&lt;tbody&gt;
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td <span class="att">align="center"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;A&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">align="center"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;B&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">align="center"</span> <span class="att">style="border-right: 1px solid ; border-bottom: 1px solid ; "</span>&gt;C&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">align="center"</span> <span class="att">style="border-bottom: 1px solid ; "</span>&gt;D&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td <span class="att">align="center"</span> <span class="att">style="border-right: 1px solid ; "</span>&gt;E&lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">align="right"</span> <span class="att">colspan="3"</span> <span class="att">style="border-right: 1px solid ; "</span>&gt;F&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                     &lt;/tbody&gt;</span>
                  &lt;/table&gt;</span>
               &lt;/div&gt;</span>
            &lt;/figure&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: informaltable.016</h2><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div></header><pre class="screen language-none line-numbers"><code>+........+........+........+........+
|   A    |   B    |   C    |   D    |
+........+........+........+........+
|   E    |                         F|
+........+........+........+........+</code></pre><p>Spanspec test.  Cell "F" should be right aligned and spanning
the last 3 columns.</p><figure id="informaltable" class="informaltable-wrapper"><div class="informaltable"><table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" align="center" width="25%" /><col class="tcol2" align="center" width="25%" /><col class="tcol3" align="center" width="25%" /><col class="tcol4" align="center" width="25%" /></colgroup><tbody><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">A</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">B</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">C</td><td style="border-bottom: 1px solid ; " align="center">D</td></tr><tr><td style="border-right: 1px solid ; " align="center">E</td><td style="border-right: 1px solid ; " colspan="3" align="right">F</td></tr></tbody></table></div></figure></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: informaltable.016</h2><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div></header><div class="content"><pre class="screen language-none line-numbers"><code>+........+........+........+........+
|   A    |   B    |   C    |   D    |
+........+........+........+........+
|   E    |                         F|
+........+........+........+........+</code></pre><p>Spanspec test.  Cell "F" should be right aligned and spanning
the last 3 columns.</p><figure id="informaltable" class="informaltable-wrapper"><div class="informaltable"><table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" align="center" width="25%" /><col class="tcol2" align="center" width="25%" /><col class="tcol3" align="center" width="25%" /><col class="tcol4" align="center" width="25%" /></colgroup><tbody><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">A</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">B</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">C</td><td style="border-bottom: 1px solid ; " align="center">D</td></tr><tr><td style="border-right: 1px solid ; " align="center">E</td><td style="border-right: 1px solid ; " colspan="3" align="right">F</td></tr></tbody></table></div></figure></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/informaltable.016.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>