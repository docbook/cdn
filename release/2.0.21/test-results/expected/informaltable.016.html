<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: informaltable.016</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: informaltable.016</h2><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div></header><pre class="screen language-none line-numbers"><code>+........+........+........+........+
|   A    |   B    |   C    |   D    |
+........+........+........+........+
|   E    |                         F|
+........+........+........+........+</code></pre><p>Spanspec test.  Cell "F" should be right aligned and spanning
the last 3 columns.</p><figure id="informaltable" class="informaltable-wrapper"><div class="informaltable"><table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" align="center" width="25%" /><col class="tcol2" align="center" width="25%" /><col class="tcol3" align="center" width="25%" /><col class="tcol4" align="center" width="25%" /></colgroup><tbody><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">A</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">B</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; " align="center">C</td><td style="border-bottom: 1px solid ; " align="center">D</td></tr><tr><td style="border-right: 1px solid ; " align="center">E</td><td style="border-right: 1px solid ; " colspan="3" align="right">F</td></tr></tbody></table></div></figure></article><script src="../../resources/js/prism.js"></script></body></html>